README.md

```diff
@@ -1 +1,53 @@
-# Capstone1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Socionics Convention
+
+> This project is about a convention where people interested in Socionics can learn about it, see real-life scenarios and examples, and meet people. Maybe you'll even meet your dual! :smile:
+
+## Built With
+
+- HTML, CSS, JS
+- Git, Github
+
+## Getting Started
+### To launch the project locally:
+#### Step 1:
+- Clone this repo locally using git in the command line and the following command
+ ```bash
+ > git clone 
+ ```
+- Alternatively, you can just download the complete zip file and extract the folder in your directory
+#### Step 2:
+- In the terminal, you can run:
+```bash
+> cd portfolio
+> code .
+```
+- Then open index.html in a live server.
+
+## Online version
+[Live Demo](https://daf98.github.io/Capstone1/)
+[Video Demo](https://www.loom.com/share/e4ac16178be64af1911ac2b9c7de2b14)
+
+## Authors
+
+👤 **Dafne Azzolina**
+
+- GitHub: [@Daf98](https://github.com/Daf98)
+- Twitter: [@dafne_azzolina](https://twitter.com/dafne_azzolina)
+- LinkedIn: [dafne-azzolina](https://www.linkedin.com/in/dafne-azzolina/)
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```

Comment:
- [ ] Kindly add an Acknowledgements to the author [Cindy Shin](https://www.behance.net/adagio07) as it's required by the [Creative Commons License](https://creativecommons.org/licenses/).as required by the [Creative Commons License](https://creativecommons.org/licenses/).
![image](https://user-images.githubusercontent.com/24703171/169136139-8b2f189e-28e2-4ef7-9fec-68827d018b84.png)

Reply:
I forgot! Thank you!